tests/BuilderWithErrors/BuilderWithErrorsTest.php

```diff
@@ -7,18 +7,20 @@
 use Doctrine\RST\Builder;
 use Doctrine\Tests\RST\BaseBuilderTest;
 
-class BuilderWithErrors extends BaseBuilderTest
+class BuilderWithErrorsTest extends BaseBuilderTest
```

Comment:
This test lacked the `Test` suffix, and so was never run and was totally broken.